src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/DynamicInsight.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    /// <summary>
+    /// Class representing Dynamic Insight
```

Comment:
Making this more descriptive about when and why to use DynamicInsights would be helpful for documentation and understanding

Reply:
I can do that. Need to push most recent changes.